beacon/sync/src/test/java/tech/pegasys/teku/beacon/sync/forward/multipeer/SyncControllerTest.java

```diff
@@ -242,6 +247,33 @@ void shouldNotNotifySubscribersWhenRunningSpeculativeTarget() {
     verify(subscriberExecutor, never()).execute(any());
   }
 
+  @Test
+  void shouldRestartSyncEverySyncAwakePeriod() {
+    final SafeFuture<SyncResult> syncResult = startFinalizedSync();
+    assertThat(syncController.isSyncActive()).isTrue();
+    syncResult.complete(SyncResult.COMPLETE);
+    assertNotSyncing();
+
+    // shouldn't restart if there are no sync targets available
```

Comment:
so basically we've called out test cases in comments, and not made them separate tests... i'd suggest both of these comments could give rise to their own test case.

Reply:
I think this is part of the preconditions of this test